src/type-translator.ts

```diff
@@ -360,6 +352,34 @@ export class TypeTranslator {
           return this.translateUnion(type as ts.UnionType);
         }
 
+        // Enum literals are represented as
+        //   ts.TypeFlags.NumberLiteral | ts.TypeFlags.EnumLiteral
+        //   ts.TypeFlags.StringLiteral | ts.TypeFlags.EnumLiteral
+        if (type.flags & ts.TypeFlags.EnumLiteral) {
+          const enumLiteralBaseType = this.typeChecker.getBaseTypeOfLiteralType(type);
+          if (!enumLiteralBaseType.symbol) {
+            this.warn(`EnumLiteralType without a symbol`);
+            return '?';
+          }
+
+          if (enumLiteralBaseType === type) {
+            // For const enums, the type and base type are the same.
+            if (type.flags & ts.TypeFlags.NumberLiteral) {
+              return 'number';
+            } else if (type.flags & ts.TypeFlags.StringLiteral) {
+              return 'string';
```

Comment:
doesn't this means we emit `@type {string}` instead of `@type {EnumMemberType}`? Those are incompatible in Closure. Why is this the `const` case?